src/components/Dashboard/AccountBanner.js

```diff
@@ -228,19 +218,11 @@ const BannerWithProbability = ({ activeBalance }) => {
   ) : (
     <Wrapper
       mainIcon={
-        probablilityTooLow ? (
-          <img
-            css={`
-              display: block;
-              margin: 0 auto;
-            `}
-            height={6 * GU}
-            src={anjSpringIcon}
-            alt=""
-          />
-        ) : (
-          <CircleGraph value={draftingProbability} size={6 * GU} />
-        )
+        <CircleGraph
+          value={probablilityTooLow ? 0.01 : draftingProbability}
+          comparisonOperator={probablilityTooLow && '<'}
```

Comment:
you mean to change the `comparisonOperator` to something like `customLabel`?

Reply:
Note: this is more a discussion for moving the component into aragonUI, I think we could merge this PR without it.

I wonder if it could be a good idea to have a `label` prop as a function returning an object, so that all the styling could be done internally. This object would contain three values:

- `value`, the actual number.
- `prefix`, the optional `<`.
- `suffix`, the `%`.

The default could be defined this way to match what we currently have:

```jsx
CicleGraph.defaultProps = {
  label(value) {
    if (value === 0) {
      return { value: '0', suffix: '%' }
    }
    if (Math.round(value * 100) < 1) {
      return { prefix: '<', value: '1', suffix: '%', }
    }
    return {
      value: String(Math.round(value * 100)),
      suffix: '%',
    }
  }
}
```

What do you guys think?

Reply:
i think that is a good approach it makes the implementation  more generic